model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -85,9 +85,21 @@ data class PackageConfiguration(
  * A matcher which matches its properties against [VcsInfo]s.
  */
 data class VcsMatcher(
+    /**
+     * The [type] to match for equality against [VcsInfo.type].
+     */
     val type: VcsType,
+    /**
```

Comment:
Please add blank lines here to separate from the previous parameter. Similar below.